src/Types.ts

```diff
@@ -706,3 +709,10 @@ export interface SubmitStrategyRequest {
 export interface SubmitStrategyResponse {
   hash: string
 }
+
+export interface requireBitUsedRequest {
+    bitmapIndex: BigIntish;
+    bit: BigIntish;
```

Comment:
can someone help me determine if it's all the required params? what i did essentially was to check the arguments of the function here: https://github.com/brinktrade/brink-strategies/blob/master/src/Primitives/Primitives01.sol#L71C12-L71C26